R/tailor.R

```diff
@@ -97,11 +97,14 @@ new_tailor <- function(type, adjustments, columns, ptype, call) {
     cli_abort("The {.arg adjustments} argument should be a list.", call = call)
   }
 
-  is_adjustment <- purrr::map_lgl(adjustments, ~ inherits(.x, "adjustment"))
+  is_adjustment <- purrr::map_lgl(adjustments, ~inherits(.x, "adjustment"))
   if (length(is_adjustment) > 0 && !any(is_adjustment)) {
     bad_adjustment <- names(is_adjustment)[!is_adjustment]
-    cli_abort("The following {.arg adjustments} do not have the class \\
-                   {.val adjustment}: {bad_adjustment}.", call = call)
+    cli_abort(
+      "The following {.arg adjustments} do not have the class \\
+                   {.val adjustment}: {bad_adjustment}.",
```

Comment:
Doesn't appear to me that air handled this reformat completely

Reply:
It's the fact that cli-abort has custom stripping of newlines and leading whitespace that makes it "feel" wrong here. In any other context, the newline and leading whitespace would be meaningful and it would be extremely important to leave them untouched. In the future this might be something a linter could look at, as we'd allow a linter to have very targeted and specific rules like "if this is a cli_abort call, look at the string and make sure its aligned if its multiline", which is more fine grained that we want to go in the formatter

See also https://github.com/posit-dev/air/issues/149#issuecomment-2587980630

Reply:
Ooo interesting, that makes sense. Thanks for the context!